packages/arb-bridge-eth/contracts/rollup/facets/RollupUser.sol

```diff
@@ -13,6 +13,17 @@ abstract contract AbsRollupUserFacet is RollupBase, IRollupUser {
         _;
     }
 
+    function blocksSpentPaused() public view override returns (uint256) {
```

Comment:
If we pause and unpause repeatedly, this will only capture the length of the last pause

Reply:
Made it so time paused accumulates through many pauses, and we can reset this timer